exporter/newrelicexporter/transformer.go

```diff
@@ -160,6 +161,26 @@ func (t *traceTransformer) SpanAttributes(span pdata.Span) map[string]interface{
 	return attrs
 }
 
+func (t *traceTransformer) SpanEvents(span pdata.Span) []telemetry.Event {
```

Comment:
Missing documentation

```suggestion
// SpanEvents transforms the recorded events of span into New Relic tracing events.
func (t *traceTransformer) SpanEvents(span pdata.Span) []telemetry.Event {
```